config.json

```diff
@@ -35,9 +35,16 @@
         {
             "name": "main_db_restore",
             "type": "named_volume",
-            "volume": "montagu_db_1",
+            "volume": "montagu_db_volume",
             "backup": false,
             "comment": "Barman takes care of backup via streaming replication"
+        },
+        {
+            "name": "barman_to_starport",
+            "type": "named_volume",
+            "volume": "montagu_db_volume",
```

Comment:
So `montagu_db_volume` is the name of the volume that barman's nightly restore writes to? I don't see a volume with this name on the annex right now, but is that because the barman task hasn't run yet?

Reply:
Rich is currently writing that task in VIMC-1710 (a blocker on this ticket). It's specifically chosen to be identical to the volume name we want once it's been restored to a real Montagu instance.